src/Pages/ContentListTable/components/AddContent/helpers.test.ts

```diff
@@ -70,3 +70,22 @@ it('mapValidationData', () => {
   expect(mapValidationData([], [])).toEqual([]);
   expect(mapValidationData(validationData, formikErrors)).toEqual(success);
 });
+
+it('Notifies on file upload failure due to size', () => {
+    const notif = jest.fn((payload) => payload);
+    const f = new File([''], 'filename', { type: 'text/html'});
```

Comment:
Ideally use a meaningful name instaed of "f". 
Although after working with Go recently I understand why you did that. lol